### PR TITLE
fix lint warnings for MapView

### PR DESCRIPTION
**Pull request by trave7er**
some more lint warning fixes. Probably the last one for today.

### CODE REVIEW

File: Libraries/Components/MapView/MapView.js [hunk 1]
```diff
@@ -14,26 +14,15 @@
...
 var resolveAssetSource = require('resolveAssetSource');
 var requireNativeComponent = require('requireNativeComponent');
 
 type Event = Object;
-type MapRegion = {
-  latitude: number;
-  longitude: number;
-  latitudeDelta: number;
-  longitudeDelta: number;
-};
```
Comment (philikon) [inline Libraries/Components/MapView/MapView.js]: This type may be used internally in FB's codebase (@fkgozali, mind checking? :)). In either case, it'd be sad to turn the clock back on the flowification of the codebase. Just use the type below somewhere so lint is happy?

Comment (fkgozali) [inline Libraries/Components/MapView/MapView.js]: @philikon: yeah, this is used in multiple places internally as an imported type.

**Comment by skevy** [comment]
@trave7er could you rebase?

**Comment by satya164** [comment]
You'll probably need to make a different PR for modifying the docs.

**Comment by trave7er** [comment]
ok, thanks @satya164 , I will close this for now, and make a new PR with docc changes.

**Comment by satya164** [comment]
@trave7er I meant just another PR with the doc changes, and leave this open. The docs and code are merged differently, as the code is synced from the internal FB repo, and the docs remain here, that's why.

**Comment by trave7er** [comment]
@satya164 , sorry for the incovenience. Unfortunately, I just did a `git push -f`, and somehow I cannot reopen this PR any more. Is it ok for me to open a new PR?

**Comment by satya164** [comment]
@trave7er Sure

**Comment by trave7er** [comment]
@satya164 I just opened a new PR: https://github.com/facebook/react-native/pull/4778

Sorry for the inconvenience.

**Pull request closed without merging**